sw/device/tests/rstmgr_cpu_info_test.c

```diff
@@ -30,42 +30,40 @@
 #define kIllegalAddr1 0x4041FFF0u
 #define kIllegalAddr2 0x40003618u
 #define kSkipComp 0x12345678u
-#define kCpuDumpSize 9
+#define kCpuDumpSize 8
```

Comment:
@jdonjdon 
can you have a look at these changes?
I think to test `current pc` and `next pc` its probably easier to create a hang situation.

Let me know what you think.